scripts/package

```diff
@@ -23,9 +23,18 @@ DAPPER_BASE_IMAGE=${REPO}/dapper-base:${TAG}
 # We expect the Dapper base image to change rarely, so pull it first
 docker pull ${REPO}/dapper-base:latest || :
 
+CHANGED_FILES_LOCAL=$(git diff --name-only)
+
+if [[ "${CHANGED_FILES_PR[@]}" =~ "package/Dockerfile.dapper-base" ]] ||
+   [[ "${CHANGED_FILES_LOCAL[@]}" =~ "package/Dockerfile.dapper-base" ]] ; then
+      echo "Dockerfile.dapper-base was modified, rebuilding dapper image"
+      docker build -t ${DAPPER_BASE_IMAGE} -f Dockerfile.dapper-base .
+      docker tag ${DAPPER_BASE_IMAGE} ${REPO}/daper-base:latest
```

Comment:
This should be `dapper-base` with two `p`s.

Reply:
Right, fixed on the other PR ':)